.travis.yml

```diff
@@ -23,7 +23,13 @@ env:
 - REPO=scripts STAGE=scripts
 
 script:
-- cd $REPO && ln -s ../omero-test-infra .omero && .omero/docker $STAGE
```

Comment:
Alternatively, should we just `cp -r ../omero-test-infra .omero` without conditional handling and put a comment indicating to review this?

Reply:
If we are happy with that, that's what I had first i.e. no conditional

Reply:
If you have the commit, I think it makes the script more manageable.